DiceKitTests/MultiplicationExpression_Tests.swift

```diff
@@ -107,12 +107,17 @@ extension MultiplicationExpression_Tests {
         
         var evaluateCalled = 0
         var stubResulter: () -> Result = { 0 }
+        var stubProbabilityMass = ProbabilityMass(0)
         
         func evaluate() -> Result {
             let result = stubResulter()
             ++evaluateCalled
             return result
         }
+        
+        var probabilityMass: ProbabilityMass {
+            return stubProbabilityMass
+        }
     }
```

Comment:
Yeah, I'm bad with the spacing within a context. At work we don't normally have a space here, that's what's catching me. I'm also not sure if I want space at the top and bottom of all types. It's definitely needed here though for consistency with the rest of our codebase for the time being.